<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlightDetailsWindow</class>
 <widget class="QWidget" name="FlightDetailsWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>%1 - flight details</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/imgs.qrc">
    <normaloff>:/mainIcon/icon.png</normaloff>:/mainIcon/icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="CallsignLabel">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::NoTextInteraction</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="AirlineLabel">
         <property name="minimumSize">
          <size>
           <width>85</width>
           <height>15</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="FromLabel">
         <property name="text">
          <string notr="true">FROM</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="indent">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ToLabel">
         <property name="text">
          <string notr="true">TO</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="indent">
          <number>10</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="FromCityLabel">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="indent">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ToCityLabel">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="indent">
          <number>10</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_18" stretch="0,2,0">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="FlightProgressWidget" name="FlightProgress" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>10</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <property name="spacing">
          <number>5</number>
         </property>
         <item>
          <widget class="QLabel" name="StdLabel">
           <property name="toolTip">
            <string extracomment="Shortcut expansion for &quot;STD&quot;">Sheduled Time of Departure</string>
           </property>
           <property name="text">
            <string notr="true">STD</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
           </property>
           <property name="margin">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="PlannedDepartureTimeLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string notr="true">1200</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignHCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="StdUtcLabel">
           <property name="text">
            <string notr="true">UTC</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
           </property>
           <property name="margin">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="spacing">
          <number>5</number>
         </property>
         <item>
          <widget class="QLabel" name="StaLabel">
           <property name="toolTip">
            <string extracomment="Shortuct expansion for &quot;STA&quot;">Sheduled Time of Arrival</string>
           </property>
           <property name="text">
            <string notr="true">STA</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
           </property>
           <property name="margin">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="PlannedArrivalTimeLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string notr="true">1200</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignHCenter</set>
           </property>
           <property name="margin">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="StaUtcLabel">
           <property name="text">
            <string notr="true">UTC</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
           </property>
           <property name="margin">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <property name="spacing">
          <number>5</number>
         </property>
         <item>
          <widget class="QLabel" name="AtdLabel">
           <property name="toolTip">
            <string extracomment="Shortcut expansion for &quot;ATD&quot;">Actual Time of Departure</string>
           </property>
           <property name="text">
            <string notr="true">ATD</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
           </property>
           <property name="margin">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="ActualDepartureTimeLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string notr="true">1200</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignHCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="AtdUtcLabel">
           <property name="text">
            <string notr="true">UTC</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
           </property>
           <property name="margin">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <property name="spacing">
          <number>5</number>
         </property>
         <item>
          <widget class="QLabel" name="EtaLabel">
           <property name="toolTip">
            <string extracomment="Shortcut expansion for &quot;ETA&quot;">Estimated Time of Arrival</string>
           </property>
           <property name="text">
            <string notr="true">ETA</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
           </property>
           <property name="margin">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="EstimatedArrivalTimeLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string notr="true">1200</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignHCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="EtaUtcLabel">
           <property name="text">
            <string notr="true">UTC</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
           </property>
           <property name="margin">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="InfoTab">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideMiddle</enum>
     </property>
     <widget class="QWidget" name="FlightDetailsTab">
      <attribute name="title">
       <string>Flight details</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="InfoLabel" name="PilotLabel" native="true"/>
       </item>
       <item>
        <widget class="InfoLabel" name="AltitudeLabel" native="true"/>
       </item>
       <item>
        <widget class="InfoLabel" name="GroundSpeedLabel" native="true"/>
       </item>
       <item>
        <widget class="InfoLabel" name="HeadingLabel" native="true"/>
       </item>
       <item>
        <widget class="InfoLabel" name="FlightPhaseLabel" native="true"/>
       </item>
       <item>
        <widget class="InfoLabel" name="ServerLabel" native="true"/>
       </item>
       <item>
        <widget class="InfoLabel" name="TimeOnlineLabel" native="true"/>
       </item>
       <item>
        <widget class="InfoLabel" name="SquawkLabel" native="true"/>
       </item>
       <item>
        <widget class="InfoLabel" name="AltimeterLabel" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="FlightPlanTab">
      <attribute name="title">
       <string>Flight plan</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="InfoLabel" name="AircraftLabel" native="true"/>
         </item>
         <item>
          <widget class="InfoLabel" name="FlightRulesLabel" native="true"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="InfoLabel" name="CruiseAltitudeLabel" native="true"/>
         </item>
         <item>
          <widget class="InfoLabel" name="TrueAirSpeedLabel" native="true"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="OriginLabel">
         <property name="text">
          <string>Origin</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="AirportDetailsButton" name="OriginButton">
         <property name="text">
          <string/>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="DestinationLabel">
         <property name="text">
          <string>Destination</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="AirportDetailsButton" name="DestinationButton">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="RouteLabel">
         <property name="text">
          <string>Route</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="RouteField">
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="RemarksLabel">
         <property name="text">
          <string>Remarks</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="RemarksField">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="TrackFlightBox">
       <property name="text">
        <string>Track this flight</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="AirportDetailsButton" name="ShowButton">
       <property name="text">
        <string>Show</string>
       </property>
       <property name="icon">
        <iconset theme="go-jump"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CloseButton">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AirportDetailsButton</class>
   <extends>QPushButton</extends>
   <header>ui/buttons/airportdetailsbutton.h</header>
  </customwidget>
  <customwidget>
   <class>FlightProgressWidget</class>
   <extends>QWidget</extends>
   <header>ui/widgets/flightprogresswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>InfoLabel</class>
   <extends>QWidget</extends>
   <header>ui/widgets/infolabel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/imgs.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>CloseButton</sender>
   <signal>clicked()</signal>
   <receiver>FlightDetailsWindow</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>444</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
